pennylane/ops/op_math/controlled_ops.py

```diff
@@ -267,7 +267,7 @@ def compute_decomposition(wires):  # pylint: disable=arguments-differ
         **Example:**
 
         >>> print(qml.CH.compute_decomposition([0, 1]))
-        [qml.RY(-0.78539816339, wires=[1]), CZ(wires=[0, 1]), qml.RY(0.78539816339, wires=[1])]
+        [qml.RY(-0.78539816339, wires=[1]), Controlled(Z(1), control_wires=[0]), qml.RY(0.78539816339, wires=[1])]
```

Comment:
Why is it now `Controlled(Z)` instead of `CZ`? This seems less intuitive to me.

Reply:
So now all controlled operations inherits the `__repr__` from `Controlled`, instead of defining their own custom names, which is why you see this.